dialogs/simplelink.js

```diff
@@ -1,7 +1,26 @@
 ﻿(function (CKEDITOR) {
     'use strict';
 
-    var containsScheme = /:\/\//;
+    /**
+     * Simple check if an URL contains a scheme.
+     *
+     * @param {String} url
+     * @return {Boolean}
+     */
+    function containsScheme (url) {
+        return /^[A-Z]+:/i.test(url);
+    }
+
+    /**
+     * Simple check if a string is an email address.
+     * Source: http://www.regular-expressions.info/email.html
+     *
+     * @param {String} str
+     * @return {Boolean}
+     */
+    function isEmail (str) {
+        return /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i.test(str);
```

Comment:
Are Norwegian characters included in this? Adding a `/u` is perhaps needed and helpful for supporting that?